src/property.html

```diff
@@ -0,0 +1,1066 @@
+            </div>
+            <!-- for info every section -->
+            <div class="grid grid-cols-2 sm:hidden sm:gap-10 xl:gap-14">
+              <!-- first -->
+              <div class="flex flex-col sm:flex-row gap-3 sm:gap-10 xl:gap-14">
+                <div class="flex flex-col gap-3 pr-3 sm:pr-0">
+                  <div class="flex flex-col gap-4">
+                    <div class="font-medium text-base text-color-addition-info">
+                      Home
+                    </div>
+                    <div
+                      class="flex flex-col gap-2 text-white font-medium text-sm"
+                    >
+                      <p>Hero Section</p>
+                      <p>Features</p>
+                      <p>Properties</p>
+                      <p>Testimonials</p>
+                      <p>FAQ’s</p>
+                    </div>
+                  </div>
+                  <div
+                    class="border-t border-solid border-color relative left-0 w-auto pr-3 sm:hidden"
+                  ></div>
+                </div>
+                <div
+                  class="flex flex-col sm:flex-row gap-3 sm:gap-10 xl:gap-14 border-r border-solid border-color sm:border-none pr-3 sm:pr-0"
+                >
+                  <div class="flex flex-col gap-4">
+                    <div class="font-medium text-base text-color-addition-info">
+                      Properties
+                    </div>
+                    <div
+                      class="flex flex-col gap-2 text-white font-medium text-sm"
+                    >
+                      <p>Portfolio</p>
+                      <p>Categories</p>
+                    </div>
+                  </div>
+                  <div
+                    class="border-t border-solid border-color relative left-0 w-auto sm:hidden"
+                  ></div>
+                  <div class="flex flex-col gap-4">
+                    <div class="font-medium text-base text-color-addition-info">
+                      Contact Us
+                    </div>
+                    <div
+                      class="flex flex-col gap-2 text-white font-medium text-sm"
+                    >
+                      <p>Contact Form</p>
+                      <p>Our Offices</p>
+                    </div>
+                  </div>
+                  <div
+                    class="border-t border-solid border-color relative left-0 w-auto sm:hidden"
+                  ></div>
+                </div>
+              </div>
+              <!-- second -->
+              <div
+                class="flex flex-col sm:flex-row gap-3 sm:gap-10 xl:gap-14 overflow-hidden"
+              >
+                <div
+                  class="flex flex-col gap-4 border-l border-solid border-color sm:border-none pl-6 sm:pl-0"
+                >
+                  <div class="font-medium text-base text-color-addition-info">
+                    About Us
+                  </div>
+                  <div
```

Comment:
Why not use ol and li tags? Make more sense!!

Reply:
![image](https://github.com/user-attachments/assets/f79d3eef-061d-4c5f-ab9a-e05302fe3db3)
actually yes